### PR TITLE
add request a built-in node page

### PR DESCRIPTION
**Pull request by ayatnkw**
### Motivation

Users frequently reach out asking support team about the process for adding a new built-in node. This update aims to provide clearer guidance, addressing common questions before they arise.

### CODE REVIEW

**Comment by StarfallProjects** [comment]
@ayatnkw  One thought: is it worth also mentioning they can use the HTTP Request node for services we don't support with nodes? I could see people hitting this page where https://docs.n8n.io/integrations/custom-operations/ would be useful.

**Comment by ayatnkw** [comment]
> @ayatnkw One thought: is it worth also mentioning they can use the HTTP Request node for services we don't support with nodes? I could see people hitting this page where https://docs.n8n.io/integrations/custom-operations/ would be useful.

Great point, I'll add this to the page :)

File: docs/integrations/builtin/request-builtin-node.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# Request a built-in node
+
+## Make a feature request
+
+The best way to request a built-in node is to post it in the n8n Community Forum. You can upvote an existing post or create a new post in the [Feature Request > Nodes category](https://community.n8n.io/c/feature-requests/nodes/){:target=_blank .external-link}.
+
+## Submit a built-in node
+
+If you've created a node you'd like to add as an official node in n8n, open a pull request on [GitHub](https://github.com/n8n-io/n8n){:target=_blank .external-link}. An n8n team member will review it. Note that new nodes take time to review and this may take some time.
```
Comment (Joffcom) [inline docs/integrations/builtin/request-builtin-node.md:15]: Can we remove this? Any new node PRs are closed with a message to direct them to comunity nodes.

File: docs/integrations/builtin/request-builtin-node.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+The best way to request a built-in node is to post it in the n8n Community Forum. You can upvote an existing post or create a new post in the [Feature Request > Nodes category](https://community.n8n.io/c/feature-requests/nodes/){:target=_blank .external-link}.
+
+## Submit a built-in node
+
+If you've created a node you'd like to add as an official node in n8n, open a pull request on [GitHub](https://github.com/n8n-io/n8n){:target=_blank .external-link}. An n8n team member will review it. Note that new nodes take time to review and this may take some time. 
+
+## Publish a community node
+
+For a faster way to make your node available, create a community node and publish it on npm. Refer to [Building community nodes](/integrations/community-nodes/build-community-nodes/) for details. Note that these nodes are available [only for self-hosted users and not for n8n Cloud users](/choose-n8n/).
```
Comment (Joffcom) [inline docs/integrations/builtin/request-builtin-node.md:19]: We could possibly add a bit to let users know that we do check community nodes and the anonymous telemetry to help us work out if a node should be added to the main set. When this happens we will reach out to the creator via email or as a github issue.

**Review by Joffcom — CHANGES_REQUESTED** [review]

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
See comment re custom operations with HTTP node

**Pull request closed without merging**